src/test/java/test/SimpleBaseTest.java

```diff
@@ -340,4 +342,14 @@ protected static File createDirInTempDir(String dir) {
 
   }
 
+  protected static void doAssert(List<String> actual, String... expectedValues) {
+    doAssert(actual, Arrays.asList(expectedValues));
+  }
+
+  protected static void doAssert(List<String> actual, List<String>  expected) {
```

Comment:
I am not sure if `AssertJ` had it or not because am not very familiar with AssertJ. I will try and revisit it and see if I can find it. If yes, then I will address this.